### ISSUE TITLE
Bug (?) with ttl and timestamp in same update

### ISSUE THREAD

**Issue by charles-crain** [outworkers/phantom#512]
The following code compiles but throws an exception:

```scala
    insert.value(_foo, thing.foo)
        .value(_.bar, thing.bar)
        .ttl(7.days)
        .timestamp(DateTime.now())
        .future()
```

The exception seems to suggest that an "AND" is not being generated in the query, i.e. we get:

    INSERT INTO table(foo, bar) VALUES(1, 2) USING TTL 123456 TIMESTAMP 7654321;

...when we should get...

    INSERT INTO table(foo, bar) VALUES(1, 2) USING TTL 123456 AND TIMESTAMP 7654321;

**Comment by alexflav23** [comment]
Hi @charles-crain,

Thank you for the bug report, that does sound like an odd issue with the DSL. We will look into it very shortly and deploy a fix.

Regards.

**Comment by alexflav23** [comment]
Hi @charles-crain,

Fixed in 1.27.0, going live after CI build hopefully succeeds.

Regards.

**Comment by charles-crain** [comment]
Thanks for the quick response!  I have a couple other issues that are more like feature requests.  I'll go ahead and post them but I am also trying to get set up so I can contribute myself as well.

**Comment by alexflav23** [comment]
Hi @charles-crain,

This was fixed in the 1.27.0 release.
